test/test.py

```diff
@@ -495,7 +500,7 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
             self.assertRaises(AttributeError, lambda: bundle.specimens[0].organ_part)
 
         # Prove that as_json returns a valid JSON structure (no cycles, correct types, etc.)
-        self.assertTrue(isinstance(json.dumps(as_json(bundle)), str))
+        self.assertTrue(isinstance(json.dumps(as_json(bundle), default=str), str))
```

Comment:
The test seems to pass for me without this change. I'm wondering what the motivation is.

Reply:
The `bundle` object now contains `datetime` objects which can't be JSON dumped without specifying the `default` argument to  `json.dumps`.
```
>>> json.dumps(datetime.datetime(1, 1, 1))
Traceback (most recent call last):
...
TypeError: Object of type datetime is not JSON serializable

>>> json.dumps(datetime.datetime(1, 1, 1), default=str)
'"0001-01-01 00:00:00"'
```
![Screen Shot 2021-05-28 at 12 09 43 PM](https://user-images.githubusercontent.com/46407464/120032518-28d28780-bfaf-11eb-874a-e4a80cc767eb.png)